apps/rush-lib/src/data/test/repo/rush.json

```diff
@@ -17,6 +17,12 @@
     "sampleEmail": "mrexample@contoso.com"
   },
 
+  "rushHooks": {
```

Comment:
> Why is it "rushHooks"?  We're already in the rush.json config file.

+1

---
In reply to: [117138465](https://github.com/Microsoft/web-build-tools/pull/196#discussion_r117138465) [](ancestors = 117138465)